certified-connectors/Team Forms/apiDefinition.swagger.json

```diff
@@ -0,0 +1,674 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Team Forms",
+    "description": "Team Forms brings digital forms into Microsoft Teams with no code.",
```

Comment:
@AnthonyPhan I would highly suggest you use the same description from your readme file.